src/js/hca/config/form.js

```diff
@@ -120,6 +120,10 @@ const formConfig = {
   version: 0,
   // Disable save in progress for production
   disableSave: __BUILDTYPE__ === 'production',
+  savedFormErrorMessages: {
```

Comment:
I kinda wish we had grouped all these config options into an object, like `saveInProgress`.